ush/wafs_grib2_0p25_blending.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+################################################################################
+#  UTILITY SCRIPT NAME :  wafs_grib2_0p25_blending.sh
+#         DATE WRITTEN :  04/02/2020
+#
+#  Abstract:  This script looks for US and UK WAFS Grib2 products at 1/4 deg,
+#             waits unblended UK data for specified period of time, and blends
+#             whenever UK data becomes available. After the waiting time window
+#             expires, the script sends out US data only if UK data doesn't arrive
+#
+#  History:  04/02/2020 - First implementation of this new script
+#            10/xx/2021 - Remove jlogfile
+#            05/25/2022 - Add ICAO new milestone Nov 2023
+#            09/08/2024 - WAFS separation
+#              - Filename changes according to EE2 standard except for files sent to UK
+#              - dbn_alert subtype is changed from gfs to WAFS
+#              - Fix bugzilla 1213: Filename should use fHHH instead of FHH.
+#              - Parallel run for each forecast hour, changed from sequential run.
+#              - Fix bugzilla 1593: Improve email notification for missing UK WAFS data.
+#              - Extend waiting time window from 15 to 25 minutes
+################################################################################
+
+set -x
+
+fhr=$1
+
+mkdir -p $DATA/$fhr
+cd "${DATA}/$fhr" || err_exit "FATAL ERROR: Could not 'cd ${DATA}/fhr'; ABORT!"
```

Comment:
The error message was missing which fhr did the cd failed.
```suggestion
cd "${DATA}/${fhr}" || err_exit "FATAL ERROR: Could not 'cd ${DATA}/${fhr}'; ABORT!"
```